include/aws/common/logging.h

```diff
@@ -58,8 +58,10 @@ enum aws_log_level {
 typedef uint32_t aws_log_subject_t;
 
 /* Each library gets space for 2^^10 log subject entries */
-#define AWS_LOG_SUBJECT_STRIDE_BITS 10
-#define AWS_LOG_SUBJECT_STRIDE (1U << AWS_LOG_SUBJECT_STRIDE_BITS)
+enum {
+    AWS_LOG_SUBJECT_STRIDE_BITS = 10,
+    AWS_LOG_SUBJECT_STRIDE = (1U << AWS_LOG_SUBJECT_STRIDE_BITS),
```

Comment:
This might be changing `AWS_LOG_SUBJECT_STRIDE` from unsigned to signed.
all enums are `int` in C
but it looks like the old `#define` was trying to be unsigned